packages/service-utils/src/core/usageV2.ts

```diff
@@ -0,0 +1,56 @@
+export interface UsageV2Event {
+  /**
+   * A unique identifier for the event. Defaults to a random UUID.
+   * Useful if your service retries sending events.
+   */
+  id?: `${string}-${string}-${string}-${string}-${string}`;
+  /**
+   * The event timestamp. Defaults to now().
+   */
+  created_at?: Date;
+  /**
+   * The source of the event. Example: "storage"
+   */
+  source: string;
```

Comment:
COULD consider enforcing the actual allowed service names here

Reply:
No strong opinion, given how infrequently this changes the product source be fine to enforce a stricter type.

Reply:
Actually I like not enforcing a type. It decouples things meaning a new service can start sending a different source right away (e.g. rpcV2) and later create a materialized view to extract/aggregate that data.